testing/kokoro/system_tests.sh

```diff
@@ -95,15 +95,15 @@ fi
 # CHANGED_DIRS will be empty when run on master.
 # Also see trampoline.sh - system_tests.sh is only run when there are
 # significant changes.
-CHANGED_DIRS=$(git --no-pager diff --name-only HEAD..master | egrep -v '(\.md$|^\.github)' | grep "/" | cut -d/ -f1 | sort -u)
+CHANGED_DIRS=$(git --no-pager diff --name-only HEAD..master | egrep -v '(\.md$|^\.github)' | grep "/" | cut -d/ -f1 | sort -u | tr '\n' ' ')
```

Comment:
Is this change intended as part of the PR?

Reply:
Yes. Before this change the for loop below didn't work because the directories were separated by newlines instead of spaces. I updated the PR title.